deploy/wip-catalog/templates/apiserver.yaml

```diff
@@ -37,11 +37,18 @@ metadata:
   name: apiserver
 spec:
 {{ if .Values.debug }}
+{{ if .Values.nodePort }}
+  type: NodePort
+{{ else }}
   type: LoadBalancer
 {{ end }}
+{{ end }}
```

Comment:
> I want a stable node port

Meaning you pick the port instead of rolling the dice and getting what you get? Got it. I'm curious though whether `LoadBalancer` still lets you specify that. I feel compelled to try. Will let you know.